base/setup/reactos/lang/ru-RU.rc

```diff
@@ -173,3 +172,12 @@ BEGIN
     IDS_PARTITION_SIZE "Размер"
     IDS_PARTITION_STATUS "Status"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_BOOTLOADER_NOINST    "Не устанавливайте"
```

Comment:
```suggestion
    IDS_BOOTLOADER_NOINST    "Не устанавливать"
```
Remaining strings I will translate separately when it's in.

Reply:
ok